.vscode/settings.json

```
@@ -8,6 +8,6 @@
     "eslint.autoFixOnSave": true,
     "editor.tabSize": 2,
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
+        "source.fixAll.eslint": "explicit"
```

Guessing this file was accidentally committed

```suggestion
        "source.fixAll.eslint": true
```

---

Whoops! Yep, unintentional. Thanks!